Week-4/Ex2.js

```diff
@@ -0,0 +1,21 @@
+const Person = function() {};
+
+Person.prototype.initialize = function(name, age) {
+    this.name = name;
+    this.age = age;
+}
+
+function Teacher() {
+    
```

Comment:
We can avoid this step

Reply:
I didn't get it. I will need to create a function to update its prototype, right?

Reply:
We can directly say var Teacher= new Person()

This will create the object and also assign prototype.
and you can extend the prototype later

Reply:
Sure, Noted. I will address these comments.